projects/ad9656_fmc/common/ad9656_bd.tcl

```diff
@@ -112,8 +113,3 @@ ad_mem_hp0_interconnect $sys_cpu_clk axi_ad9656_rx_xcvr/m_axi
 ad_mem_hp2_interconnect $sys_dma_clk sys_ps7/S_AXI_HP1
 ad_mem_hp2_interconnect $sys_dma_clk axi_ad9656_rx_dma/m_dest_axi
 
-# interrupts
-
-ad_cpu_interrupt ps-10 mb-15 axi_ad9656_rx_jesd/irq
```

Comment:
The interrupts are necessary, they just don't need to be discontinuous

Reply:
Re-added interrupts in v5